homeassistant/components/roku/media_player.py

```diff
@@ -59,12 +59,14 @@ def __init__(self, host):
         self.ip_address = host
         self.channels = []
         self.current_app = None
+        self._available = False
         self._device_info = {}
         self._power_state = "Unknown"
 
     def update(self):
         """Retrieve latest state."""
         try:
+            self._available = True
```

Comment:
This should be the last line in this try block.
You only know it is available again after the requests have been done successfully.